rest/route/host_spawn.go

```diff
@@ -66,6 +66,20 @@ func (hph *hostPostHandler) Run(ctx context.Context) gimlet.Responder {
 		if err := CheckUnexpirableHostLimitExceeded(ctx, user.Id, hph.env.Settings().Spawnhost.UnexpirableHostsPerUser); err != nil {
 			return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "checking expirable host limit"))
 		}
+
+		var defaultTimeZone string
+		if hph.options.TimeZone != "" {
+			defaultTimeZone = hph.options.TimeZone
+		} else if user.Settings.Timezone != "" {
+			defaultTimeZone = user.Settings.Timezone
+		}
+		if !hph.options.HasSchedule() {
+			// If making an unexpirable host and the request did not specify a
+			// sleep schedule, set it to the default sleep schedule to ensure it
+			// sets some sleep schedule.
+			hph.options.SleepScheduleOptions = host.GetDefaultSleepSchedule(defaultTimeZone)
```

Comment:
Same comment/question for here, but also should we include this comment in the other one too? Would it be worth having a method on the options that we call so the logic doesn't separate?

Reply:
See above. The defaulting behavior should be a little clearer now since there's two defaulting methods.